### DISCUSSION TITLE
Is it possible to execute two "special commands"

### DISCUSSION THREAD

**Discussion by camroth89** [complexlogic/flex-launcher#46] [Q&A] [answered]
I am trying to make it so that when I put the PC to sleep and then wake it up again, Flex Launcher will be back on the main menu instead of the system menu.

The :back command works to go back to the main menu, but I cannot seem to use both :back and :sleep for the same item.

Is it possible to do this? 

Thanks.

**Comment by complexlogic** [comment, marked as answer]
This is currently not possible. I've had the same idea before though, because it would be nice to return to the main menu after waking from sleep. I considered making a config setting `PostSleepCmd`, or something along those lines. But as of now, it's unimplemented.

**Reply by camroth89** [reply]
Ah ok. Thanks for the answer.

Somewhat related, my workaround is to use a batch file to kill Flex Launcher, start Flex Launcher, and sleep the PC.

However, for some reason whenever I start Flex Launcher using a .bat it launches, but the icons are all invisible. Items can be opened, but there's no icon.

Any idea why that may be?

**Reply by complexlogic** [reply]
Most likely the working directory changes when you run the batch script. The default Flex Launcher Windows config uses relative paths, e.g. `.\assets\icons\some_icon.png`. You could convert these to absolute paths, e.g. `C:\path\to\some_icon.png`. That will make it so that your config doesn't depend on the working directory.

**Reply by camroth89** [reply]
Ah duh, that seems obvious now that you say it. 

Thanks!
